src/components/Nav/Nav.js

```diff
@@ -0,0 +1,57 @@
+import React from 'react';
+import '../Nav/Nav.scss';
+
+const Nav = () => {
+  return (
+    <nav>
```

Comment:
```suggestion
    <nav className="nav">
```
로 컨벤션 맞춰주시면 더 좋을 것 같습니다.